### PR TITLE
1382987 update content

### PR DESCRIPTION
**Pull request by Mahesh2516**
Updated document with a new section. Please review

### CODE REVIEW

**Review by laujan — CHANGES_REQUESTED** [review]

File: msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md [hunk 1]
```diff
@@ -204,5 +204,11 @@ Please warn users before your app downloads any files or executable (`.exe`  )in
 * The message extension `@mention` executables should be clear, easy to understand, and easy to read.
 ![Message extension](../../../../assets/images/faq/message-extension.png)
 
+### M365 Publisher Attestation
+#### Complete the Publisher Attestation in Partner Center
+* Refer to the documentation for more details about the [attestation program](https://docs.microsoft.com/microsoft-365-app-certification/docs/attestation).
```
[the remark this answers is not in the document]
Comment (KirtiPereira) [reply msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md]: Hi @Mahesh2516. Please do the changes to the file path as discussed. Also, please incorporate the points shared by Priya for this requirement. Thank you.

[the remark this answers is not in the document]
Comment (Mahesh2516) [reply msteams-platform/concepts/deploy-and-publish/appsource/prepare/frequently-failed-cases.md]: @laujan and @KirtiPereira, I have added the points and corrected the links. Please review.

**Review by KirtiPereira — COMMENTED** [review]
@Mahesh2516 - please change the label to ready-to-merge.

**Comment by Mahesh2516** [comment]
All updates have been done and it is ready-to-merge.

**Review by laujan — APPROVED** [review]

**Pull request merged**